### PR TITLE
fix(select): missing option becoming available shows wrong label

### PR DESCRIPTION
**Pull request by saithis**
With mode 'id' when a missing options becomes available the label didn't update to the one of the
loaded item. Also the items emitted twice in this case.

### CODE REVIEW

File: projects/components/select/src/defaults/default-select-data-source.ts [hunk 1]
```diff
@@ -101,44 +99,32 @@ export class DefaultPsSelectDataSource<T = any> extends PsSelectDataSource<T> {
     const loadedOptions$ = optionsLoadTrigger$.pipe(
       switchMap(() => this._loadData()),
       startWith<PsSelectItem<T>[]>([]),
+      map(options => options.map(normalizeLabel)),
       shareReplay({ bufferSize: 1, refCount: true })
     );
 
     // Values die nicht in den Options sind, generieren wir hier
```
Comment (6rayWolf) [inline projects/components/select/src/defaults/default-select-data-source.ts:106]: this comment should be in english ;)

File: projects/components/select/src/select-data.component.ts [hunk 2]
```diff
@@ -114,8 +114,8 @@ export class PsSelectDataComponent<T = any> implements AfterViewInit, OnDestroy
   }
 
   /** Die Fehlernachricht, wenn es beim Items Laden einen Fehler gab */
```
Comment (6rayWolf) [inline projects/components/select/src/select-data.component.ts:116]: these comments should also be in english i guess?

File: projects/components/select/src/select.component.spec.ts [hunk 3]
```diff
@@ -201,16 +204,8 @@ describe('PsSelectComponent', () => {
     // expect(component.emittedValues).toEqual([undefined]);
```
Comment (6rayWolf) [inline projects/components/select/src/select.component.spec.ts:204]: remove?

File: projects/components/select/src/defaults/default-select-data-source.ts (hunk 1 above)
Comment (saithis) [reply projects/components/select/src/defaults/default-select-data-source.ts]: done

File: projects/components/select/src/select-data.component.ts (hunk 2 above)
Comment (saithis) [reply projects/components/select/src/select-data.component.ts]: done

File: projects/components/select/src/select.component.spec.ts (hunk 3 above)
Comment (saithis) [reply projects/components/select/src/select.component.spec.ts:204]: no, this is only commented out until the bug in MatSelect is fixed

**Pull request merged**